trailblazer/store/crud/update.py

```diff
@@ -256,15 +256,16 @@ def update_analysis(
         analysis: Analysis = self.get_analysis_with_id(analysis_id)
 
         if comment:
-            LOG.info(f"Adding comment {comment} to analysis {analysis.family}")
+            LOG.info(f"Adding comment {comment} to analysis {analysis.id}")
```

Comment:
Do we have anything better in Trailblazer to identify analyses with than their integer id?

Reply:
Well, using the case name it is associated with is misleading since we have many analyses per case. I think the database id for the case is the most reasonable identifier at the moment